Assets/__Scripts/MapEditor/AudioTimeSyncController.cs

```diff
@@ -175,19 +200,38 @@ private void OnDestroy()
         CurrentSeconds = offsetMS;
     }
 
+    public IEnumerator StopPlayingDelayed(float delaySeconds)
+    {
+        stopScheduled = true;
+        yield return new WaitForSeconds(delaySeconds);
+        if (IsPlaying) TogglePlaying();
+        stopScheduled = false;
```

Comment:
Won't this always be set by TogglePlaying?

Reply:
Currently, yes.
But if it were to be called when not playing it could result in stopScheduled staying true.
That being said I could probably move it up a line to prevent StopCoroutine being called unnecessarily.